app/src/main/java/xyz/jhughes/laundry/adapters/MachineAdapter.java

```diff
@@ -21,28 +22,37 @@
 
 import butterknife.Bind;
 import butterknife.ButterKnife;
+import retrofit2.Call;
+import retrofit2.Callback;
+import retrofit2.Response;
 import xyz.jhughes.laundry.LaundryParser.Constants;
 import xyz.jhughes.laundry.LaundryParser.Machine;
 import xyz.jhughes.laundry.LaundryParser.MachineStates;
 import xyz.jhughes.laundry.MachineFilter;
+import xyz.jhughes.laundry.apiclient.MachineService;
+import xyz.jhughes.laundry.notificationhelpers.ScreenOrientationLockToggleListener;
+import xyz.jhughes.laundry.notificationhelpers.OnMachineChangedToInUse;
 import xyz.jhughes.laundry.R;
 import xyz.jhughes.laundry.SnackbarPostListener;
 import xyz.jhughes.laundry.analytics.AnalyticsHelper;
 import xyz.jhughes.laundry.notificationhelpers.NotificationCreator;
+import xyz.jhughes.laundry.runnables.MachineCheckerRunnable;
 import xyz.jhughes.laundry.storage.SharedPrefsHelper;
 
 public class MachineAdapter extends RecyclerView.Adapter<MachineAdapter.ViewHolder> {
 
     private final String roomName;
     private final SnackbarPostListener listener;
+    private final ScreenOrientationLockToggleListener lockListener;
```

Comment:
Can you change this variable name to something like 

`mOnOrientationLockLictener`